provider/examples/resources/asa/asav/wait_for_asa.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Post provisioning is commented out at the minute. If it stays commented out we can delete this